docs/en/schema.core.rst

```diff
@@ -770,7 +770,8 @@ A Contact is an object with the following properties:
    must not be obfuscated. To improve resistance against e-mail
    collection, use ``\x64`` to replace ``@``, as allowed by the YAML
    specification.
--  ``phone`` - phone number (with international prefix)
+-  ``phone`` - phone number (with international prefix). This has to be
+   a string delimited by double quotes. 
```

Comment:
I would remove the requirement of it being delimited by double quotes, it is an implementation detail and not the only way to properly represent strings in yaml